docs/core/components/viewer.component.md

```diff
@@ -61,7 +61,8 @@ See the [Custom layout](#custom-layout) section for full details of all availabl
 | Name | Type | Default value | Description |
 | ---- | ---- | ------------- | ----------- |
 | allowFullScreen | `boolean` | true | Toggles the 'Full Screen' feature. |
-| allowGoBack | `boolean` | true | Allows `back` navigation |
+| allowGoBack | `boolean` | true | Allows `back` navigation. |
+| closeButtonPosition | `string` | `left` | Set close button position right/left. |
```

Comment:
instead of string should be CloseButtonPosition type

Reply:
Done